lib/hammer_cli_katello/repository.rb

```diff
@@ -436,12 +435,14 @@ def update_content_upload(upload_id, repo_id, file)
       end
 
       def import_uploads(uploads, opts = {})
+        rpm_type = option_content_type
         publish_repository = opts.fetch(:last_file, false)
         sync_capsule = opts.fetch(:last_file, false)
         params = {:id => get_identifier,
                   :uploads => uploads,
                   publish_repository: publish_repository,
-                  sync_capsule: sync_capsule
+                  sync_capsule: sync_capsule,
+                  content_type: rpm_type,
```

Comment:
Not sure why we need the `rpm_type` variable instead of just passing in `option_content_type`.